vars/provisionNodes.groovy

```diff
@@ -220,6 +256,10 @@ EOF'''
       if (inst_rpms) {
          provision_script += '''\nyum -y erase ''' + inst_rpms
       }
+      provision_script += '\nrpm --import /etc/pki/rpm-gpg/*'
```

Comment:
Why do we trust these keys?  We should only install keys that come from a trustable source, be that the secure web server of a known and trustable entity or a web server controlled by us that we have populated the keys into only after verifying their authenticity.

These keys are the linchpin to the trustability of the entire system since they are what are validating the RPMs being installed.  If we fail to be diligent at this point, the whole system comes crumbling down.

Reply:
Changed to import keys from local site that we control.